charts/ziti-controller/templates/alt-certificate.yaml

```diff
@@ -20,9 +20,9 @@ spec:
   {{- end }}
   issuerRef:
     {{- toYaml $cert.issuerRef | nindent 4 }}
-  {{- if gt (len $cert.dnsNames) 0 }}
+  {{- if gt (len $cert.altDnsNames) 0 }}
```

Comment:
My goal here is more consistent names to distinguish Ziti's DNS SANs from its alternative DNS SANs. 

I may roll this back since it's a breaking change. I was thinking "Maybe no one is using it" but how can we know?